backend/src/modules/mobile/mobile.controller.ts

```diff
@@ -132,6 +137,44 @@ export class MobileController {
     return user;
   }
 
+  @Post('/lock')
```

Comment:
if this is a toggle can we make it clear in the endpoint like `toggleLock` or `lock/toggle`